packages/generator-clay-component/app/index.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * © 2018 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+'use strict';
+
+let _ = require('lodash');
+let chalk = require('chalk');
+let yeoman = require('yeoman-generator');
+let yosay = require('yosay');
+
+module.exports = yeoman.generators.Base.extend({
+	initializing: function() {
```

Comment:
Can we assume this will be running in the context of a recent enough Node version that we can use ES6 object concise methods? ie.

```js
module.exports = yeoman.generators.Base.extend({
        initializing() {
                // etc...
```